docs/LOCAL_DEV.md

```diff
@@ -60,6 +60,78 @@ open htmlcov/index.html
 ```
 
 ### To run localization
+
```

Comment:
@hsong-rh I think we can remove this whole section since its already properly configured now in the defaults.py